PHPCompatibility/Docs/Classes/NewFinalClassConstantsStandard.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<documentation title="New final class constants">
+    <standard>
+    <![CDATA[
+    Class constants can be declared as final since PHP 8.1.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Cross-version compatible: not using the final modifier.">
+        <![CDATA[
+class Foo {
+    const BAR = 10;
+}
+        ]]>
+        </code>
+        <code title="PHP >= 8.1: using the final modifier.">
+        <![CDATA[
+class Foo {
+    <em>final</em> const BAR = 10;
```

Comment:
The HTML tag appears literally in markdown docs. I recommend avoiding markdown in code samples. Other than that, everything looks good on the docs side.

```suggestion
    final const BAR = 10;
```

Reply:
The `<em>` markers are intended to highlight the "issue" in the HTML view.
In the `Text` and the `MarkDown` view, they are automatically removed when the report is generated.

How did you generate the report to get them to show ?

Reply:
Just checked the source code of the generators which are included with PHPCS and the above is correct. Also tested with the docs for another sniff I was just working on.

Reply:
This is the generator code I mean: https://github.com/squizlabs/PHP_CodeSniffer/blob/51335eb46b2b940b6c429643fe96f514d4a4e4a1/src/Generators/Markdown.php#L131-L141

If the code in the phpcs docs repo does not line up with that, it will need to be adjusted.